[IMP] inventory: adjust manual val for v.16 (#4091)
odoo/documentation: content/applications/inventory_and_mrp/inventory/management/reporting/using_inventory_valuation.rst

In Odoo, this process can be conducted manually— by warehouse employees physically counting the
products— or automatically through the database.

Automatic inventory valuation
=============================

To view automatically generated entries on product reception, the inventory valuation method must be
set as automated on the *product category*. To do so, navigate to the :menuselection:`Product
Categories` list by going to :menuselection:`Inventory app --> Configuration --> Product Categories`
and select the desired product category. On the form, set the :guilabel:`Inventory Valuation` as
:guilabel:`Automated` and the :guilabel:`Costing Method` to any of the three selection.

.. seealso::
   :ref:`How inventory valuation is set <inventory/management/reporting/inventory_valuation_config>`

In order to understand how moving products in and out of stock affects the company's overall value,
consider the following product and stock moves scenario below.

Receive a product
-----------------

To automatically calculate the value of the incoming product, a simple *table*, go to the product

> form and click the :guilabel:`➡️ (External link)` icon next to the :guilabel:`Product Category`

Reviewer: Update icon text to be consistent throughout the doc
```suggestion
form and click the :guilabel:`➡️ (arrow)` icon next to the :guilabel:`Product Category`
```

Update product unit price
-------------------------

For any company: lead times, supply chain failures, and other risk factors can contribute to
invisible costs. Although Odoo attempts to accurately represent the stock value, *manual valuation*
serves as an additional tool to update the unit price of products.

.. important::
   Manual valuation is intended for products that can be purchased and received for a cost greater
   than 0, or have product categories set with :guilabel:`Costing Method` set as either
   :guilabel:`Average Cost (AVCO)` or :guilabel:`First In First Out (FIFO)`.

.. image:: using_inventory_valuation/add-manual-valuation.png
   :align: center
   :alt: Add manual valuation of stock value to a product.

Create manual valuation entries on the :guilabel:`Stock Valuation` dashboard by first navigating to
:menuselection:`Inventory app --> Reporting --> Valuation`. Next, to enable the *product
revaluation* feature, select :menuselection:`Group by --> Product` to organize all the records by
product. Click on the gray :guilabel:`▼ (drop-down)` icon to reveal stock valuation line items

> below, as well as a teal :guilabel:`+ (plus)` button on the right.

Reviewer 2: Can use emoji here too if you want
```suggestion
below, as well as a teal :guilabel:`➕ (plus)` button on the right.
```

Click the teal :guilabel:`+ (plus)` button to open up the :guilabel:`Product Revaluation` form.
Here, the inventory valuation for a product can be recalculated, by increasing or decreasing the
unit price of each product.

.. image:: using_inventory_valuation/product-revaluation.png
   :align: center
   :alt: Product revaluation form adding a value of $1.00 with the reason being inflation.

.. note::

>    This button is only visible after grouping entries by product.

Reviewer 2: Admonition block before the image.
Also specify what *this* is.

Inventory valuation journal entries
-----------------------------------

In Odoo, automatic inventory valuation records are also recorded in the :menuselection:`Accounting
app --> Accounting --> Journal Entries`. On this comprehensive list of accounting entries, inventory

> valuation entries are identified by checking values in the :guilabel:`Journal` column, or looking

Reviewer 2: Suggestion: "entries" back to back dilutes its meaning
```suggestion
valuation records are identified by checking values in the :guilabel:`Journal` column, or looking
```

for the :guilabel:`Reference` column value which matches the warehouse operation reference (e.g.
`WH/IN/00014` for receipts).

Clicking on an inventory valuation journal entry opens a *double-entry accounting* record. These
records are generated by Odoo to track the change of value in inventory valuation as products are
moved in and out of the warehouse.

.. example::
   To view the inventory valuation of 10 *tables*, costing $10.00 each, upon reception from the
   vendor, go to the :menuselection:`Journal Entries` page found in :menuselection:`Accounting app
   --> Accounting --> Journal Entries`. Here, click the journal line where the :guilabel:`Reference`
   column value matches the reference on the receipt, `WH/IN/00014`.

   .. image:: using_inventory_valuation/stock-valuation-product.png
      :align: center
      :alt: Stock valuation page depicting the products within a shipment.

   *Stock interim* is a holding account for money intended to pay vendors for the product. The

>    *stock valuation* account stores the value of all on-hand stock.

Reviewer 2: If we're pointing to / describing UI elements, then these get guis

Author: This one is tough. These are journal accounts that Odoo creates by default, but they can be changed.
![image](https://github.com/odoo/documentation/assets/23611711/7b96dc6e-c47f-4727-9972-2766847a9d70)
What do you think, Zac? `stock interim` or *stock interim* ?

Reviewer 2: Would do backticks `stock interim` since it's a field value 👍

products— or automatically through the database.

.. seealso::
   :ref:`Inventory valuation methods <inventory/management/reporting/inventory_valuation_config>`

Inventory valuation basics
==========================

In order to understand how moving products in and out of stock affects the company's overall value,
consider the following product and stock moves scenario below.

Receive a product
-----------------

> For example, consider a physical product, a simple *table*, which is categorized as `Office
> Furniture` in the product form's :guilabel:`Product Category` field.

Reviewer: Also with this text - if this is a selected value from a dropdown it should be guilabel, otherwise user input as literal
```suggestion
For example, consider a physical product, a simple *table*, which is categorized as :guilabel:`Office
Furniture` in the product form's :guilabel:`Product Category` field.
```

> Navigate to the the product category itself by going to :menuselection:`Inventory app -->

Reviewer: Double word "the"
```suggestion
Navigate to the product category itself by going to :menuselection:`Inventory app --> Configuration
```

> Configuration --> Product Categories`. On the form, set the :guilabel:`Costing Method` as `First In
> First Out (FIFO)` and :guilabel:`Inventory Valuation` as `Automated`.

Reviewer: If the field values "First In First Out" and "Automated" are _selects_ from a dropdown, then they should have :guilabel:s, otherwise if they are user input keep them as literal formatting 🙂
```suggestion
Configuration --> Product Categories`. On the form, set the :guilabel:`Costing Method` as :guilabel:`First In
First Out (FIFO)` and :guilabel:`Inventory Valuation` as :guilabel:`Automated`.
```

.. tip::
   Alternatively access the :guilabel:`Product Categories` dashboard by clicking on the

>    :guilabel:`( ➡️ ) arrow` icon on the individual product's form, by hovering over the

Reviewer: Slightly corrected icon formatting :)
```suggestion
   :guilabel:`➡️ (arrow)` icon on the individual product's form, by hovering over the
```

   :alt: Decreased stock valuation after a product is shipped.

.. _inventory/reporting/valuation-report:

Inventory valuation report
==========================

To view the current value of all products in the warehouse, first turn on :ref:`Developer mode
<developer-mode>` and navigate to :menuselection:`Inventory app --> Reporting --> Valuation`. The
page displays detailed records of products with the :guilabel:`Date`, :guilabel:`Quantity`,
:guilabel:`Unit Value`, and :guilabel:`Total Value` of the inventory.

.. important::
   :ref:`Developer mode <developer-mode>` **must** be enabled to see the :guilabel:`Valuation`
   option under :guilabel:`Reporting`.

.. image:: using_inventory_valuation/inventory-valuation-products.png
   :align: center
   :alt: Inventory valuation report showing multiple products.

The :guilabel:`Valuation At Date` button, located in the top-left corner of the :guilabel:`Stock
Valuation` page, reveals a pop-up window. In this pop-up, the inventory valuation of products
available during a prior specified date can be seen and selected.

.. tip::
   View a detailed record of a product's inventory value, stock move, and on-hand stock by selecting

>    the teal :guilabel:`( ➡️ ) arrow` button to the right of the :guilabel:`Reference` column value.

Reviewer: Icon formatting again
```suggestion
   the teal :guilabel:`➡️ (arrow)` button to the right of the :guilabel:`Reference` column value.
```

Update product unit price
-------------------------

For any company: lead times, supply chain failures, and other risk factors can contribute to

> invisible costs. Although Odoo attempts to accurately represent the stock value, *manual valuation*

Reviewer: Same with the field values here below on lines 108-109 - if they are from a dropdown please use :guilabels:, otherwise user input use keep it as literal

   smart button.

.. tip::
   The :doc:`consignment </applications/inventory_and_mrp/inventory/management/misc/owned_stock>`
   feature allows ownership to items in stock. Thus, products owned by other companies are not
   accounted for in the host company's inventory valuation.

Deliver a product
-----------------

In the same logic, when a table is shipped to a customer and leaves the warehouse, the stock
valuation decreases. The :guilabel:`Valuation` smart button on the :abbr:`DO (Delivery Order)`,
likewise, displays the stock valuation record as it does on a :abbr:`PO (Purchase Order)`.

.. image:: using_inventory_valuation/decreased-stock-valuation.png
   :align: center
   :alt: Decreased stock valuation after a product is shipped.

.. _inventory/reporting/valuation-report:

Inventory valuation report
==========================

To view the current value of all products in the warehouse, first turn on :ref:`Developer mode
<developer-mode>` and navigate to :menuselection:`Inventory app --> Reporting --> Valuation`. The
page displays detailed records of products with the :guilabel:`Date`, :guilabel:`Quantity`,

> :guilabel:`Unit Value` and :guilabel:`Total Value` of the inventory.

Reviewer 3: ```suggestion
:guilabel:`Unit Value`, and :guilabel:`Total Value` of the inventory.
```

------------------------------

To view the current value of all products in the warehouse, go to :menuselection:`Inventory app -->

> Reporting --> Valuation`. The page displays detailed records of products with the :guilabel:`Date`,
> :guilabel:`Quantity`, :guilabel:`Unit Value` and :guilabel:`Total Value` of the inventory.

Reviewer 3: ```suggestion
Reporting --> Valuation`. Doing so, reveals a page displaying detailed records of products with the
:guilabel:`Date`, :guilabel:`Quantity`, :guilabel:`Unit Value` and :guilabel:`Total Value` of the
inventory.
```
**make sure this doc mentions that this Reporting menu option is only available in 'Developer Mode' -

> .. important::
>    :ref:`Developer mode<developer-mode>` must be turned on to see the :guilabel:`Valuation` option
>    under :guilabel:`Reporting`.

Reviewer 3: ```suggestion
.. important::
   :ref:`Developer mode<developer-mode>` **must** be turned on to see the :guilabel:`Valuation`
   option under :guilabel:`Reporting`.
```
added ** to emphasize the importance of this step

this is a great note - but perhaps it could be more useful if its incorporated into the navigation steps, as it took me a second to realize I needed to be in Developer Mode before my eyes found this notation - just a suggestion. totally your call.

.. image:: using_inventory_valuation/inventory-valuation-products.png
   :align: center
   :alt: Inventory valuation report showing multiple products.

> The :guilabel:`Inventory At Date` button, located in the top-left corner of the :guilabel:`Stock
> Valuation` dashboard, shows the inventory valuation of products available during a prior specified
> date.

Reviewer 3: ```suggestion
The :guilabel:`Valuation At Date` button, located in the top-left corner of the :guilabel:`Stock
Valuation` page, reveals a pop-up window. In this pop-up, the inventory valuation of products
available during a prior specified date can be selected and seen.
```
the name of the button in my v16 runbot was "Valuation At Date" (not "Inventory At Date") - added the mention of the pop-up window that appears, as well

.. tip::

>    View a detailed record of a product's inventory value, stock move, and on-hand stock by selecting
>    the teal ":guilabel:`-->`" button to the right of the *Reference* column value.

Reviewer 3: ```suggestion
   View a detailed record of a product's inventory value, stock moves, and on-hand stock by
   clicking the teal arrow icon (:guilabel:`-->`) to the right of the :guilabel:`Reference` column
   value.
```
added :gui because its the title of a column/visible on screen

>    the teal ":guilabel:`-->`" button to the right of the *Reference* column value.

Author: *Reference* or :guilabel:`Reference`?

   :alt: Add manual valuation of stock value to a product.

Create manual valuation entries on the :guilabel:`Stock Valuation` dashboard by first navigating to

> :menuselection:`Inventory app --> Reporting --> Valuation`. Next, to enable the product revaluation

Reviewer: Italicizing and capitalizing the *Product Revaluation* button because we are referring to a UI element outside of the active instruction
```suggestion
:menuselection:`Inventory app --> Reporting --> Valuation`. Next, to enable the *Product Revaluation*
```

button, select :menuselection:`Group by --> Product` to organize all the records by product. Click

> on the gray drop-down icon to reveal stock valuation line items below, as well as a teal

Reviewer: I believe this "dropdown icon" would require a :guilabel: since we are instructing the user to click on it and describing it visually
```suggestion
on the gray :guilabel:`drop-down` icon to reveal stock valuation line items below, as well as a teal
```

Author: You're right about the guilabel! What do you think about taking it further and turning it into an emoji?
```suggestion
on the gray :guilabel:`🔽 (drop-down)` icon to reveal stock valuation line items below, as well as a teal
```

> on the gray drop-down icon to reveal stock valuation line items below, as well as a teal

Reviewer: Alternatively you could also add the icon found [here](https://www.symbolcopy.com/keyboard-symbols.html)
```suggestion
on the gray :guilabel:`▼ (drop-down)` icon to reveal stock valuation line items below, as well as a teal
```

> :guilabel:`( + ) plus` button on the right.

Reviewer: Icon formatting
```suggestion
:guilabel:`+ (plus)` button on the right.
```

> Click the teal :guilabel:`( + ) plus` button to open up the :guilabel:`Product Revaluation` form.

Reviewer: ```suggestion
Click the teal :guilabel:`+ (plus)` button to open up the :guilabel:`Product Revaluation` form.
```

This button is only visible after grouping entries by product. Here, updates to the calculation of
inventory valuation can be made, by increasing or decreasing the unit price of each product.

.. image:: using_inventory_valuation/product-revaluation.png
   :align: center
   :alt: Product revaluation form adding a value of $1.00 with the reason being inflation.

Inventory valuation journal entries
-----------------------------------

In Odoo, automatic inventory valuation records are also recorded in the :menuselection:`Accounting
app --> Accounting --> Journal Entries`. On this comprehensive list of accounting entries,
identify the inventory valuation entries by looking for the :guilabel:`Reference` column value which
matches the warehouse operation reference (e.g. `WH/IN/00014` for receipts).

Clicking on the desired inventory valuation journal entry opens a *double-entry accounting* record
that Odoo generated to track the value of products as they move in and out of the warehouse.

.. example::
   To view the inventory valuation of 10 *tables*, costing $10.00 each, upon reception from the

>    vendor, go to the :guilabel:`Journal Entries` page found in :menuselection:`Accounting app -->

Reviewer: Please use menuselection for page names, especially when part of a navigation 🙂
```suggestion
   vendor, go to the :menuselection:`Journal Entries` page found in :menuselection:`Accounting app -->
```

   :alt: Add manual valuation of stock value to a product.

Create manual valuation entries on the :guilabel:`Stock Valuation` dashboard by first navigating to
:menuselection:`Inventory app --> Reporting --> Valuation`. Next, to enable the product revaluation
button, select :menuselection:`Group by --> Product` to organize all the records by product. Click

> on the gray drop-down icon to then reveal stock valuation line items below as well as a teal
> :guilabel:`+ (plus)` button on the right.

Reviewer 3: ```suggestion
on the gray drop-down icon to reveal stock valuation line items below, as well as a teal
:guilabel:`+ (plus)` button on the right.
```

Click the teal :guilabel:`+ (plus)` button to open up the :guilabel:`Product Revaluation` form. This

> button is only visible after grouping entries by product.
> where updates to the calculation of inventory valuation can be made, by increasing or decreasing the
> unit price of each product.

Reviewer 3: ```suggestion
button is only visible after grouping entries by product. Here, updates to the calculation of
inventory valuation can be made, by increasing or decreasing the unit price of each product.
```

   :alt: Add manual valuation of stock value to a product.

Create manual valuation entries on the :guilabel:`Stock Valuation` dashboard by first navigating to

> :menuselection:`Inventory app --> Reporting --> Valuation`. Then, select :menuselection:`Group by
> --> Product` to organize all the records by product. Click on the little gray drop-down icon to then
> reveal stock valuation line items below as well as a teal :guilabel:`+ (plus)` button on the right.

Reviewer 3: ```suggestion
:menuselection:`Inventory app --> Reporting --> Valuation`. Next, select :menuselection:`Group by
--> Product` to organize all the records by product. Then, click on the gray drop-down icon, on the
far left of the desired line item, to reveal stock valuation line items and a teal :guilabel:`+
(plus)` clickable icon on the right.
```
